kotlin/android/app/src/main/java/dev/firezone/android/features/settings/ui/AdvancedSettingsFragment.kt

```diff
@@ -67,20 +70,74 @@ class AdvancedSettingsFragment : Fragment(R.layout.fragment_settings_advanced) {
                     requireActivity().finish()
 
                 is SettingsViewModel.ViewAction.FillSettings -> {
-                    binding.etAuthBaseUrlInput.apply {
-                        setText(action.config.authBaseUrl)
+                    binding.etAuthUrlInput.apply {
+                        setText(action.config.authUrl)
                     }
+
                     binding.etApiUrlInput.apply {
                         setText(action.config.apiUrl)
                     }
+
                     binding.etLogFilterInput.apply {
                         setText(action.config.logFilter)
                     }
+
+                    applyManagedStatus(action.managedStatus)
                 }
             }
         }
     }
 
+    private fun applyManagedStatus(status: ManagedConfigStatus) {
```

Comment:
Unfortunately it wasn't very straightforward to add a toast for a disabled text input. We have to show a clickable info icon here.